### PR TITLE
created creator landing page and added base components to it

### PR DESCRIPTION
**Pull request by migui3230**
lmk if there are any changes you want in here especially with how the box looks and the fonts on the page

### CODE REVIEW

**Review by MikaelCarpenter — COMMENTED** [review]
One other thing to mention, the way Next routing works is that they use the filename (case-sensitive) for the route. So right now this route would be `localhost:3000/creatorLandingPage/`. I would suggest changing the filename to be "kebab-case" because that's more typical for domain routes. i.e. `/creator-landing-page/`

**Comment by migui3230** [comment]
image

File: app/pages/creatorLandingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { NextPage } from "next";
+import NameHeader from "./components/NameHeader";
+import CreatorDescription from "./components/CreatorDescription";
+import BenefitBox from "./components/BenefitBox";
```
[the remark this answers is not in the document]
Comment (migui3230) [reply app/pages/creatorLandingPage.tsx]: @MikaelCarpenter do you want me to delete those component files after pulling them in? Also do you know what `Nextpage` is used for?

**Comment by migui3230** [comment]
image

new page after changes

File: app/pages/creatorLandingPage.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MikaelCarpenter) [reply app/pages/creatorLandingPage.tsx]: Yea go ahead and delete them. And I saw you moved the component definition into the page file, but you can just not make them components at all. Like all of the JSX should just be a part of `CreatorLandingPage` component

I'm pretty sure `NextPage` is just a TypeScript type that they give you, but you don't have to use it

**Comment by MikaelCarpenter** [comment]
Looking at the merge conflicts, you're going to want to rebase this off of the most recent `main` branch. It'll get you the latest changes which already included the `@tailwindcss/typography` module and added it to the config

Here's some docs on what a rebase is: https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase

They make it sound all complicated but you'll just want to do:
```bash
# make sure you have the latest main branch
git fetch

# rebase on top of the latest main branch
git rebase origin/main

# The above command will likely pause and tell you to resolve merge conflicts
# After you adjust the code to remove those conflicts you'll
git add <path to file you touched>
git rebase --continue

# Repeat until rebase is complete
```

**Comment by NBNARADHYA** [comment]
@migui3230 Please fix the linting changes.

I think it exists mainly in `yarn.lock`. You could first undo changes, then do `yarn add package1 package2 ...`. Also keep the format on save to false if you are using vs code.

**Review by NBNARADHYA — CHANGES_REQUESTED** [review]
@migui3230 Please fix the linting changes.

I think it exists mainly in `yarn.lock`. You could first undo changes, then do `yarn add package1 package2 ...`. Also keep the format on save to false if you are using vs code.

**Review by MikaelCarpenter — APPROVED** [review]
Looks good !

**Pull request merged**